source/dotnet/Library/AdaptiveCards.Rendering.Wpf/RenderedAdaptiveCard.cs

```diff
@@ -20,6 +20,10 @@ public RenderedAdaptiveCard(FrameworkElement frameworkElement, AdaptiveCard orig
 
         internal Dictionary<string, Func<object>> InputBindings { get; set; } = new Dictionary<string, Func<object>>();
 
+        /// <summary>
+        /// If any modifications were done to the rendered card they will be reported here
```

Comment:
I get why you call them "modifications", but that just feels odd to me. I think "Any warnings encountered while rendering will be reported here" makes more sense. For example, I don't really see a custom renderer missing as a "modification", I see that as a "warning", as in "Hey, I'm warning you that we couldn't render an element"